### PR TITLE
pyenv-virtualenv: add activate/deactivate examples

### PR DESCRIPTION
**Pull request by kintoandar**
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/pyenv-virtualenv.md [hunk 1]
```diff
@@ -10,3 +10,11 @@
 - List all existing virtual environments:
 
 `pyenv virtualenvs`
+
+- Activate a virtual environment:
+
+`pyenv activate {{my-virtual-env-3.6.6}}`
```
Comment (zdroid) [inline pages/common/pyenv-virtualenv.md]: Maybe `{{my-virtual-env-3.6.6}}` should be changed here, as well as the one in the first example.

Comment (kintoandar) [reply pages/common/pyenv-virtualenv.md]: Agree, pinning a version in an example does not age well. May I suggest replacing `my-virtual-env-3.6.6` with `example`?

Comment (ivanhercaz) [reply pages/common/pyenv-virtualenv.md]: Or even `virtualenv_name`.

Comment (schneiderl) [reply pages/common/pyenv-virtualenv.md]: I agree with @ivanhercaz, `virtualenv_name` will do.

Comment (kintoandar) [reply pages/common/pyenv-virtualenv.md]: Changed 👍

**Comment by ivanhercaz** [comment]
Thank you for the fix, @kintoandar. Now, after you applied the suggestion made by @zdroid, it looks good to me, but it is necessary he review it again to approve it.

Just a note, you have not to make a force push, because then, in cases like the edits of one page, we squash and merge it in just one commit with the title of the PR. So don't worry about it.

Again, thank you very much for your contribution.

**Pull request merged**